cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -700,7 +700,7 @@ script "js_instance_cost_mapping", type:"javascript" do
         idle_instance.push({
           "accountId": ds_get_caller_identity[0]['account']
```

Comment:
Since we're standardizing these fields, should we use `accountID` in these policies, matching the capitalization in `resourceID`?
```suggestion
          "accountID": ds_get_caller_identity[0]['account']
```